src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -1239,6 +1250,19 @@ internal override bool IsDeclaredReadOnly
             }
         }
 
+        internal override bool IsInitOnly
+        {
+            get
+            {
+                if (!_packedFlags.IsInitOnlyPopulated)
+                {
+                    bool isInitOnly = ReturnTypeWithAnnotations.CustomModifiers.HasIsInitOnlyModifier();
+                    _packedFlags.InitializeIsInitOnly(isInitOnly);
+                }
+                return _packedFlags.IsInitOnly;
```

Comment:
>return _packedFlags.IsInitOnly [](start = 16, length = 30)

We shouldn't return true for anything, but a "set" accessor. It doesn't look like this method guarantees that.